code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -462,14 +462,26 @@
 		//Monkestation addition start: this is a port of #77651 which was closed, so I'm putting this as an addition
 	else if(helper.zone_selected == BODY_ZONE_PRECISE_MOUTH) //Boops
 		if(HAS_TRAIT(src, TRAIT_BADTOUCH) && prob(75))
-			helper.visible_message(span_notice("[src] matrix dodges [helper]'s boop, holy shit!"), span_notice("[src] matrix dodges your boop, holy shit!"))
+			helper.visible_message(span_notice("[src] matrix dodges [helper]'s boop, holy shit!"), \
+						null, span_hear("You hear a strange noise, like someone fighting for their life!"), DEFAULT_MESSAGE_RANGE, list(helper, src))
+			to_chat(helper, span_notice("[src] matrix dodges your boop, holy shit!"))
+			to_chat(src, span_notice("[helper] tried to boop you but you avoid it with a matrix dodge, holy shit!"))
 		else if(istype(get_item_by_slot(ITEM_SLOT_MASK), /obj/item/clothing/mask/gas/clown_hat))
 			playsound(src, 'sound/items/bikehorn.ogg', 50, TRUE)
-			helper.visible_message(span_notice("[helper] honks [src]'s nose"), span_notice("You honk [src]'s nose."))
+			helper.visible_message(span_notice("[helper] honks [src]'s nose"), \
+						null, span_hear("You hear honk!"), DEFAULT_MESSAGE_RANGE, list(helper, src))
```

Comment:
feels like it should be "you hear a honk" instead of "you hear honk"
```suggestion
						null, span_hear("You hear a honk!"), DEFAULT_MESSAGE_RANGE, list(helper, src))
```

Reply:
oh yea I forgot about that one too lmao